config.ts

```diff
@@ -14,15 +14,16 @@ export const HOPR_TOKEN_USED_CONTRACT_ADDRESS = wxHOPR_TOKEN_SMART_CONTRACT_ADDR
 
 // Safe Contracts
 export const SAFE_SERVICE_URL = 'https://safe-transaction.stage.hoprtech.net';
-export const HOPR_NODE_STAKE_FACTORY = '0xa2e2F71f687914f5DC2010632bC2dEbB1B9FC1D5';
-export const HOPR_NODE_MANAGEMENT_MODULE = '0xb3B09FA6504F9d59F1a9740F68f3E904ca857A82';
-export const HOPR_NODE_SAFE_REGISTRY = '0x3E7c4720934ff6A9FE122Cb761f36a11E9b848D9';
-export const HOPR_NETWORK_REGISTRY = '0x2f3243adC9805F6dd3E01C9E9ED31675A5B16902';
+export const HOPR_NODE_STAKE_FACTORY = '0x098B275485c406573D042848D66eb9d63fca311C';
+export const HOPR_NODE_MANAGEMENT_MODULE = '0xB7397C218766eBe6A1A634df523A1a7e412e67eA';
+export const HOPR_NODE_SAFE_REGISTRY = '0xe15C24a0910311c83aC78B5930d771089E93077b';
+export const HOPR_NETWORK_REGISTRY = '0x582b4b586168621dAf83bEb2AeADb5fb20F8d50d';
 
 //Subgraphs
 export const STAKE_SUBGRAPH = 'https://api.studio.thegraph.com/query/40439/hopr-stake-all-seasons/v0.0.10';
-export const STAKING_V2_SUBGRAPH = 'https://api.studio.thegraph.com/query/40438/subgraph-dufour/version/latest';
-//export const STAKING_V2_SUBGRAPH = 'https://api.studio.thegraph.com/proxy/40438/test-dufour/version/latest';
+export const STAKING_V2_SUBGRAPH = 'https://api.studio.thegraph.com/query/40439/hopr-nodes-dufour/version/latest';
```

Comment:
I see that this url is for dufour dev environment, but the contracts are production contracts. Does this subgraph index does contracts as well?

Reply:
My bad, this subgraph works

Reply:
It should work fine